### PR TITLE
Better docs

### PR DESCRIPTION
**Pull request by luandro**
- Added instructions to build translations and run the Javascript bundler prior to building for Android on contributing docs
- Added instructions for downloading and preparing offline maps wth the needed structure

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -58,6 +58,15 @@ icons/
 
 ### Offline Maps
 
+Use [mapbox-style-downloader](https://github.com/digidem/mapbox-style-downloader) to download styles, tiles, glyphs, and sprites for offline use.
+
+To create an asar file from the tiles:
```
Comment (gmaclennan) [inline README.md]: Actually, this is missing from the docs in mapbox-style-downloader: There is an `--asar` option for the downloader that will do that for you. Can you add that to the downloader docs?

File: README.md [hunk 2]
```diff
@@ -58,6 +58,15 @@ icons/
 
 ### Offline Maps
 
+Use [mapbox-style-downloader](https://github.com/digidem/mapbox-style-downloader) to download styles, tiles, glyphs, and sprites for offline use.
```
Comment (gmaclennan) [inline README.md]: Maybe link to the npm page, and add instructions to install globally with `npm install --global`.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -100,7 +100,19 @@ Android Studio](https://developer.android.com/studio/run/managing-avds). Choose
...
+npm run build:translations
+```
+
+Connect your phone with USB, or start up the emulator. Then start the Javascript bundler:
+
+```sh
+npm start
+```
+
+On another terminal window build and run the
```
Comment (gmaclennan) [inline CONTRIBUTING.md]:
```suggestion
In another terminal window build and run the
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -100,7 +100,19 @@ Android Studio](https://developer.android.com/studio/run/managing-avds). Choose
...
+Build translations with:
+
+```sh
+npm run build:translations
+```
+
+Connect your phone with USB, or start up the emulator. Then start the Javascript bundler:
+
+```sh
+npm start
```
Comment (gmaclennan) [inline CONTRIBUTING.md:112]: The `npm run android` command should open a new terminal with the bundler if it is not already running, however that might not work on all machines, so I agree with your proposed changes here, which are more reliable.

**Review by gmaclennan — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 4 above)
Comment (luandro) [reply CONTRIBUTING.md:112]: Yea, doesn't do that on my machine.

File: README.md [hunk 5]
```diff
@@ -58,22 +58,45 @@ icons/
...
+Use `mapbox-style-downloader` to download styles, tiles, glyphs, and sprites for offline use:
+
+```sh
+mapbox-style download mapbox://styles/mapbox/streets-v9 \
+  -a \
+  --token='MAPBOX_API_TOKEN' \
+  -o styledir \
+  -z 8 \
+  -Z 13 \
+  -b '-60.1364 1.5626 -58.0627 3.475'
```
Comment (gmaclennan) [inline README.md:80]: Add flag `--asar`

**Review by gmaclennan — APPROVED** [review]

**Pull request merged**